CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+## [3.5.5.1] - 2021-05-xx
```

Comment:
Is it intended that the day is missing?

Reply:
Yes, I was going to fill it once the PR was reviewed.